Page Object/Book/loginAPI.ts

```diff
@@ -0,0 +1,95 @@
+import { baseAPI } from "../baseAPI";
+export class LoginApi extends baseAPI {
+    private readonly baseUrl: string;
+    constructor (page) {
+        super (page)
+        this.baseUrl = 'https://demoqa.com';
+    }
+
+    async registerUserRequest(userName: string, password: string) {
+        const response = await this.page.request.post(`${this.baseUrl}/Account/v1/User`, {
+            data: {"userName":userName,"password":password}
+            });
+            const body = await this.GetResponceBody(response)
```

Comment:
methods are usually named with a small letter + there is a small spelling error. You can install extension `Code Spell Checker` in VSCode in order to avoid spelling errors